src/env-inl.h

```diff
@@ -303,7 +303,17 @@ inline Environment::Environment(IsolateData* isolate_data,
   v8::HandleScope handle_scope(isolate());
   v8::Context::Scope context_scope(context);
   set_as_external(v8::External::New(isolate(), this));
-  set_binding_cache_object(v8::Object::New(isolate()));
+
+  v8::Local<v8::Primitive> Null = v8::Null(isolate());
+  v8::Local<v8::Object> binding_cache_object = v8::Object::New(isolate());
+  CHECK(binding_cache_object->SetPrototype(context, Null).FromJust());
+  set_binding_cache_object(binding_cache_object);
+
+  v8::Local<v8::Object> internal_binding_cache_object =
+    v8::Object::New(isolate());
```

Comment:
Style: four space indent (or write it as `auto var = ...` so it fits on one line.)

Reply:
done